CHANGELOG.md

```diff
@@ -139,7 +139,7 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
   respond to GET requests for the health check only.
   
 - `kafkaexporter, kafkareceiver`: Add a new mechanism "AWS_MSK_IAM_OAUTHBEARER" for kafka exporter and kafka receiver. This mechanism use the AWS MSK IAM SASL Signer for Go https://github.com/aws/aws-msk-iam-sasl-signer-go. (#19747)
```

Comment:
Some projects don't like changing changelogs. If this is such a project, I can drop these changes.

Alternatively, if it's just a matter of adding a `Skip Changelog` label, that's fine. Or splitting those changes into a distinct PR as they're mostly unrelated to most of the other changes.

Reply:
just make a separate PR for that one file

Reply:
- https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37129